pypeit/core/gui/identify.py

```diff
@@ -80,6 +79,7 @@ def __init__(self, canvas, axes, spec, specres, detns, line_lists, par, lflag_co
             Spatial ID corresponding to slit
         wv_calib : :obj:`dict`, None, optional
             If a best-fitting solution exists, and you wish to load it, provide the wv_calib dictionary.
+        pxtoler : float, optional
```

Comment:
Please, add the description for `pxtoler`

Reply:
done